css-pseudo-4/Overview.bs

```diff
@@ -1147,6 +1147,20 @@ Tree-Abiding Pseudo-elements</h2>
   non-inheritable properties take their <a>initial values</a> as usual.
   [[CSS-CASCADE-4]]
 
+<h3 id=partlike>
+Part-Like Pseudo-Elements</h3>
+
+  A subset of [=tree-abiding pseudo-elements=],
+  the <dfn>part-like pseudo-elements</dfn>,
+  have slightly stronger requirements:
+  they act as if they have a well-defined location in the document tree.
+  This enables them to interact with some other platform features
+  as if they were real elements.
```

Comment:
What's an example of this, other than their full participation in CSS layout?

Reply:
Like being used in `exportparts` attribute.